cfn.yaml

```diff
@@ -133,7 +140,7 @@ Resources:
       MemorySize: 1536
       Role: !GetAtt ExecutionRole.Arn
       Runtime: java8
-      Timeout: 60
+      Timeout: 120
 
   DataLakeAlertsWorkerLambda:
```

Comment:
I wondered whether it might be worth adding [`ReservedConcurrentExecutions`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-function.html#cfn-lambda-function-reservedconcurrentexecutions) here, just to guard against mistakes which could cause this function to become very greedy! Any opinions on that would be much appreciated @alexduf?

Reply:
The problem with `ReservedConcurrentExecution` is that it also removes that count from the global pool of lambdas available for that account. So I would advise against it in this case.

It's usually used as a way to allocate a slice of the account pool to a specific app. I find that property a bit clumsy at best.

Reply:
Ah okay - thanks for the explanation @alexduf